From e55a351fdf3d48a89abe7ffe2fe614b1a57dde74 Mon Sep 17 00:00:00 2001
From: Sylvain Pichon <sylvainx.pichon@intel.com>
Date: Thu, 22 Mar 2012 11:09:40 +0100
Subject: [PATCH 04/49] [PORT FROM MAIN] FM: add FM mode in sytem/core audio.h

BZ: 20378

Change-Id: If6cde142e07902576c31a359641c26d4aa0cda25
Orig-Change-Id: Ie75e40bc6e60055cce036b5dcb2f7748d59465ed
Orig-Change-Id: I6df47eb80ec12208ba72b6a4d3456e660f0e605c
Signed-off-by: Sylvain Pichon <sylvainx.pichon@intel.com>
Reviewed-on: http://android.intel.com:8080/75600
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Barthes, FabienX <fabienx.barthes@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/system/audio.h |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/include/system/audio.h b/include/system/audio.h
index d246070..d09910b 100644
--- a/include/system/audio.h
+++ b/include/system/audio.h
@@ -270,6 +270,13 @@ typedef enum {
 } audio_mode_t;
 
 typedef enum {
+    AUDIO_MODE_FM_OFF,
+    AUDIO_MODE_FM_ON,
+    AUDIO_MODE_FM_CNT,
+    AUDIO_MODE_FM_MAX = AUDIO_MODE_FM_CNT -1
+} audio_fm_mode_t;
+
+typedef enum {
     AUDIO_IN_ACOUSTICS_AGC_ENABLE    = 0x0001,
     AUDIO_IN_ACOUSTICS_AGC_DISABLE   = 0,
     AUDIO_IN_ACOUSTICS_NS_ENABLE     = 0x0002,
-- 
1.7.9

